### ISSUE TITLE
No line-up

### ISSUE THREAD

**Issue by Czral** [Loki-Afro/metalarchives#20]
Hello there,

When I try to fetch the line-up for a band (Current line-up, Last known line-up, Past line-up or Live line-up) I don't get any results (Size is 0). I tried even well known bands such as Metallica or Iron Maiden. At the same time I get the line-up for albums perfectly fine.

Thank you for your time!

**Comment by Loki-Afro** [comment]
yeah .. had a quick look, there is not a single test thats doing anyting with lineup, and this crappy html "parsing" broke here

`com.github.loki.afro.metallum.core.parser.site.helper.band.MemberParser#parse` which should be rewritten entirely i think using jsoup as the rest of the code does
